src/refsys/coordinates.py

```diff
@@ -36,18 +36,7 @@
 import numpy as np
 
 # Own Libs
-
-
-#######################################################################
-
-
-#######################################################################
-# Constants area
-#######################################################################
-
-# Eccentric anomaly constants
-MAX_ITER_E = 15
-EPSILON_E = 1e-10
+import src.constants as constants

```

Comment:
Can we change it to `as cts`? :P

Reply:
I think the current way would be more explicit

Reply:
Ok, nevermind